build/csharpTranspiler.ts

```diff
@@ -1204,6 +1204,7 @@ class NewTranspiler {
             content = this.regexAll (content, [
                 [/object  = functions;/g, '' ], // tmp fix
                 [/assert/g, 'Assert'],
+                [ /object exchange(?=[,)])/g, 'Exchange exchange' ],
```

Comment:
base tests were missing this regex, for live-tests & "main test" file that regex is used https://github.com/ttodua/ccxt/blob/afterconstruct-in-init/build/csharpTranspiler.ts#L1254